<?xml version="1.0" ?>
<launch>
	<arg name="output" default="screen"/>
	
  <!-- Launch the tracking node -->
  <node pkg="visp_auto_tracker" type="visp_auto_tracker" name="visp_auto_tracker">
    <param name="model_path" value="$(find qr_detector_pkg)/resource/visp_qr_models" />
    <param name="model_name" value="pattern" />
    <param name="debug_display" value="False" />
    <remap from="/visp_auto_tracker/camera_info" to="/tarot/camera_raw/camera_info"/>
    <remap from="/visp_auto_tracker/image_raw" to="/tarot/camera_raw/image_raw"/>
  </node>

  <node pkg="qr_detector_pkg" type="qr_detector_exe" name="qr_detector_exe" output="$(arg output)">
  </node>


</launch>
